### PR TITLE
[HackerRank] The Blacklist

### LINKED ISSUE #44: The Blacklist Problem [Difficulty-Advanced][DP-Hackerrank]
I would like to contribute the solution for [The Blacklist Problem](https://www.hackerrank.com/challenges/the-blacklist/problem) of HackerRank.

Difficulty level- **Advanced**
Algorithm Section:- **Dynamic Programming**

Please Assign!!

### PR DESCRIPTION
**Pull request by dhruvhacks**
closes #44

### CODE REVIEW

**Comment by aneels3** [comment]
@dhruvhacks please read the [Contribution Guidelines](https://github.com/aneels3/Algorithm/blob/master/CONTRIBUTION.md) and Change the PR name

**Comment by dhruvhacks** [comment]
@aneels3 It's already changed!

File: HackerRank Problem/Python/Blacklist.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+for already_killed in range(0, N):
+  for mask in range(0, MASK):
+    for hitman in range(0, K):
+      if hitman_available(hitman, mask):
+        mask_after = use_hitman(hitman, mask)
+        for num_to_kill in range(1, N - already_killed+1):
+          dp[mask_after][num_to_kill + already_killed] = min(
+            dp[mask_after][num_to_kill + already_killed],
+            dp[mask][already_killed] + sum(cost[hitman][already_killed:already_killed+num_to_kill]))
```
Comment (aneels3) [inline HackerRank Problem/Python/Blacklist.py]: I can see that your code is not indented properly. Please use 4 spaces for the Indentation.

Comment (dhruvhacks) [reply HackerRank Problem/Python/Blacklist.py]: Sure. I personally use 2-spaces in cases of higher complexities. Updating this in a min. :+1:

**Comment by dhruvhacks** [comment]
@aneels3 Indentation updated

**Pull request merged**